missing_mga/missing_methods.py

```diff
@@ -107,8 +110,8 @@ def missing_value_heatmap(self):
         Creates a heatmap to visualize the distribution of missing values in the dataset.
         """
         plt.figure(figsize=(10, 6))
-        sns.heatmap(self._obj.isnull(), cmap='viridis', cbar=False)
-        plt.title('Missing Values Heatmap')
+        sns.heatmap(self._obj.isnull(), cmap="viridis", cbar=False)
+        plt.title("Missing Values Heatmap")
```

Comment:
Do you like double quotes better? The simple ones for me! Jaja 😶‍🌫️

Reply:
jajaja my code formatter is running on save!